CHANGELOG.md

```diff
@@ -1,59 +1,213 @@
+  [#13436](https://github.com/rust-lang/cargo/pull/13436)
 - Clarify the `version` field in `[package]` is optional in Cargo.toml
   [#13390](https://github.com/rust-lang/cargo/pull/13390)
 - Improve "Registry Authentication" docs
   [#13351](https://github.com/rust-lang/cargo/pull/13351)
+- Improve "Specifying Dependencies" docs
+  [#13341](https://github.com/rust-lang/cargo/pull/13341)
+- Remove `package.documentation` from the “before publishing” list.
+  [#13398](https://github.com/rust-lang/cargo/pull/13398)
 
 ### Internal
 
+- 🎉 Integrated tracing-chrome as a basic profiler for Cargo itself.
+  [docs](https://doc.crates.io/contrib/tests/profiling.html)
+  [#13399](https://github.com/rust-lang/cargo/pull/13399)
+  [#13551](https://github.com/rust-lang/cargo/pull/13551)
+- Updated to `gix` 0.58.0
+  [#13380](https://github.com/rust-lang/cargo/pull/13380)
+- Updated to `git2` 0.18.2
+  [#13412](https://github.com/rust-lang/cargo/pull/13412)
+- Updated to `jobserver` 0.1.28 
+  [#13419](https://github.com/rust-lang/cargo/pull/13419)
+- Updated to `supports-hyperlinks` 3.0.0
+  [#13511](https://github.com/rust-lang/cargo/pull/13511)
+- Updated to `rusqlite` 0.31.0
+  [#13510](https://github.com/rust-lang/cargo/pull/13510)
+- bump-check: use symmetric difference when comparing source code
+    [#13581](https://github.com/rust-lang/cargo/pull/13581)
+- bump-check: include rustfix and cargo-util-schemas
+    [#13421](https://github.com/rust-lang/cargo/pull/13421)
 - ci: enable m1 runner
   [#13377](https://github.com/rust-lang/cargo/pull/13377)
+- ci: Ensure lockfile is respected during MSRV testing via `cargo-hack`.
+  [#13523](https://github.com/rust-lang/cargo/pull/13523)
+- cargo-util-schemas: Consistently compare MSRVs via `RustVersion::is_compatible_with`.
+  [#13537](https://github.com/rust-lang/cargo/pull/13537)
 - console: Use new fancy `anstyle` API
   [#13368](https://github.com/rust-lang/cargo/pull/13368)
+  [#13562](https://github.com/rust-lang/cargo/pull/13562)
 - fingerprint: remove unnecessary Option in `Freshness::Dirty`
   [#13361](https://github.com/rust-lang/cargo/pull/13361)
+- fingerprint: abstract `std::fs` away from on-disk index cache
+  [#13515](https://github.com/rust-lang/cargo/pull/13515)
+- mdman: Updated to `pulldown-cmark` 0.10.0
+  [#13517](https://github.com/rust-lang/cargo/pull/13517)
+- refactor: Renamed `Config` to `GlobalContext` 
+  [#13409](https://github.com/rust-lang/cargo/pull/13409)
+  [#13486](https://github.com/rust-lang/cargo/pull/13486)
+  [#13506](https://github.com/rust-lang/cargo/pull/13506)
+- refactor: Removed unused `sysroot_host_libdir`.
+  [#13468](https://github.com/rust-lang/cargo/pull/13468)
+- refactor: Expose source/spans to Manifest for emitting lints
+  [#13593](https://github.com/rust-lang/cargo/pull/13593)
+- refactor: Flatten manifest parsing 
+  [#13589](https://github.com/rust-lang/cargo/pull/13589)
```

Comment:
This includes a fix so that we print warnings for virtual manifests, see https://github.com/rust-lang/cargo/pull/13589/commits/714359289d8c66c5f652127db2d1b8292e9cda57

Reply:
I see. This is like a side effect of the refactor. Do you suggest make it more visible and repharse as "Print warnings from parsing virtual manifests"? Any other suggestion?

Reply:
That or list both